src/flags.rs

```diff
@@ -58,7 +59,8 @@ pub fn print_usage() {
 -D or --log-debug  Log debug output.
 -h or --help       Print this message.
 --v8-options       Print V8 command line options.
---deps             Print module dependencies."
+--deps             Print module dependencies.
+--types            Print global environment types."
```

Comment:
Maybe better: `print typescript declarations for the deno standard library`.
cc @ry

Reply:
"Print runtime typescript declarations."

Reply:
`Print the type declarations for the runtime environment.`?

Reply:
Crossed the streams... I will go with what @ry suggested.